Velocity4LumiContainers/images/rccl/specs.yaml

```diff
@@ -0,0 +1,5 @@
+versions:
+  - spec: 85eb1f1
+dependencies:
+  - spec: rocm
+    when: rocm
```

Comment:
What does it mean to just set `when: rocm` without a version specification? Isn't that just the same as leaving out the `when` clause?